### PR TITLE
Feature connect api

### PR DESCRIPTION
**Pull request by cdsandoval**

### CODE REVIEW

File: src/views/productList.jsx [hunk 1]
```diff
@@ -2,45 +2,17 @@
 import React from "react";
 import Product from "../components/product";
 import { jsx } from "@emotion/core";
-import { Link } from "@reach/router";
```
Comment (byteconde) [inline src/views/productList.jsx]: Well done, you deleted fake data. Do you know there's an alternative way to have this? I've just discover that there's an alternative way to generate fake JSON data: [https://www.json-generator.com/](click here)

File: src/views/Order.js [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
 
 function Order({
   description = "descripcion",
   name = "name",
   price = "price"
 }) {
   const cartProducts = useSelectorCart();
-  console.log(cartProducts);
+  const user = useUser();
+  if (!user.name) return <Redirect to="/login" noThrow />;
```
Comment (byteconde) [inline src/views/Order.js]: Well done, noThrow avoid us to see unnecessary views with React erros.

File: src/views/productList.jsx (hunk 1 above)
Comment (cdsandoval) [reply src/views/productList.jsx]: Thank you, I did not know anything about this website. Looks great, going to give a look.

File: src/views/Order.js (hunk 2 above)
Comment (cdsandoval) [reply src/views/Order.js]: Thank you :+1:

**Pull request merged**